app/models/classification.rb

```diff
@@ -98,11 +98,16 @@ def ns
   # rubocop:disable Style/NumericPredicate
 
   def self.classify(obj, category_name, entry_name, is_request = true)
-    cat = Classification.lookup_by_name(category_name, obj.region_id)
-    unless cat.nil?
-      ent = cat.find_entry_by_name(entry_name, obj.region_id)
-      ent.assign_entry_to(obj, is_request) unless ent.nil? || obj.is_tagged_with?(ent.to_tag, :ns => "none")
-    end
+    cat = Classification.find_by_name(category_name, obj.region_id)
```

Comment:
this does feel like ui code.

also programmatically, it is difficult to know if this succeeded or failed without string compare.

nice thing about this code is that it doesn't blow up our other classes that assume this method will not throw an exception.